core/templates/dev/head/pages/exploration_player/audio_bar_directive.html

```diff
@@ -26,10 +27,27 @@
 
 <style>
```

Comment:
Can you add `audio-bar` before all these CSS classes? Otherwise these rules will match stuff everywhere in the app and not just within this directive.

(It was also incorrect before, unfortunately.)

Reply:
@seanlip Done!